Assets/Play Tests/Views/AssetViewerTests.cs

```diff
@@ -28,18 +35,18 @@ public IEnumerator PlacesAsset()
                 .FindWithTag("EventSystem")
                 .GetComponent<AssetViewer>();
             
-            aViewer.placeInScene(_desiredAsset);
+            aViewer.placeInScene(assets[0]);
```

Comment:
What are your thoughts on making a local variable called `desiredAsset` that loads from `array[0]` instead?

Reply:
That would bring back some readability, I'll do that.